pkg/duffle/manifest/create.go

```diff
@@ -1,11 +1,12 @@
 package manifest
 
 import (
-	"encoding/json"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 
+	"github.com/docker/go/canonical/json"
```

Comment:
nit: Should the order of the two imports be switched?

Reply:
I don't know what the expected order is anymore. I basically reordered until the linter stopped whining.